experimental/examples/bubblebench/editable-shared-tree/src/schema.ts

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    brand,
+    EditableTree,
+    FieldKinds,
+    fieldSchema,
+    namedTreeSchema,
+    rootFieldKey,
+    SchemaData,
+    ValueSchema,
+} from "@fluid-internal/tree";
+
+export const stringSchema = namedTreeSchema({
+    name: brand("String"),
+    value: ValueSchema.String,
+});
+
+export const numberSchema = namedTreeSchema({
+    name: brand("number"),
+    value: ValueSchema.Number,
+});
+
+export const iBubbleSchema = namedTreeSchema({
+    name: brand("Test:BubbleBenchAppStateiBubble-1.0.0"),
+    localFields: {
+        x: fieldSchema(FieldKinds.value, [numberSchema.name]),
+        y: fieldSchema(FieldKinds.value, [numberSchema.name]),
+        r: fieldSchema(FieldKinds.value, [numberSchema.name]),
+        vx: fieldSchema(FieldKinds.value, [numberSchema.name]),
+        vy: fieldSchema(FieldKinds.value, [numberSchema.name]),
+    },
+});
+
+export const iClientSchema = namedTreeSchema({
+    name: brand("Test:BubbleBenchAppStateiClient-1.0.0"),
+    localFields: {
+        clientId: fieldSchema(FieldKinds.value, [stringSchema.name]),
+        color: fieldSchema(FieldKinds.value, [stringSchema.name]),
+        bubbles: fieldSchema(FieldKinds.sequence, [iBubbleSchema.name]),
+    },
+});
+
+export const AppStateSchema = namedTreeSchema({
+    name: brand("Test:BubbleBenchAppState-1.0.0"),
+    localFields: {
+        clients: fieldSchema(FieldKinds.sequence, [iClientSchema.name]),
```

Comment:
the root of the document is a field, so rather than having a AppStateSchema in that field, you could just make this sequence of clients the `rootAppStateSchema` and delete AppStateSchema.

Reply:
If I were to delete AppStateSchema and make rootappStateSchema  into `export const rootappStateSchema = fieldSchema(FieldKinds.sequence, [clientSchema.name]);`

then how would Iy define that as a field under the name of "clients"?

Reply:
if the list of clients is the root of the document, there is no need for a field named "clients".

Reply:
1. How do I access a sequence on a root node when it is not assigned to a field with editable tree ? `tree[EmptyKey]` ?

2. Am I correct in assuming the JsonableTree representation of the root node would then be:
```
{
    type: AppStateSchema.name,
    fields: {
        [EmptyKey]: []
    },
};
```

Reply:
No. When you delete  AppStateSchema and use a sequence of clients at the root, you no longer need to add any default tree in initializeTree, since the tree already has an empty root sequence. You can just delete the:

```
        const initialRootNodeJson = {
            type: appStateSchema.name,
            fields: {
                clients: [],
            },
        };
        tree.context.root.insertNodes(0, [initialRootNodeJson].map(singleTextCursor));
```

If you wanted to keep the insert logic (and there is no need to): what you would do instead is:

```
        tree.context.root.insertNodes(0, []);
```

That does nothing (it inserts a an empty sequence into the root, which has no effect). Thats all you need since you want to start with an empty sequence of clients. This is why you no longer need any tree initialization at all.


You also want to delete `AppStateTreeProxy` and replace use of it in AppState.ts with `ClientTreeProxy[] & EditableField`